### PR TITLE
Update logo and commonize it

### PR DESCRIPTION
**Pull request by foostan**
I created the `corne` logo.

corne-logo

So I updated the logo of OLED screen.

before

before

after

after

### CODE REVIEW

File: keyboards/crkbd/keymaps/default/rules.mk [hunk 1]
```diff
@@ -4,18 +4,19 @@
...
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
 MIDI_ENABLE = no            # MIDI controls
 AUDIO_ENABLE = no           # Audio output on port C6
 UNICODE_ENABLE = no         # Unicode
 BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
-RGBLIGHT_ENABLE = yes        # Enable WS2812 RGB underlight.  Do not enable this with audio at the same time.
-SWAP_HANDS_ENABLE = no        # Enable one-hand typing
+RGBLIGHT_ENABLE = yes       # Enable WS2812 RGB underlight.  Do not enable this with audio at the same time.
+SWAP_HANDS_ENABLE = no      # Enable one-hand typing
+LOCAL_GLCDFONT = no         # use each keymaps "glcdfont.c" insted of "keymaps/default/glcdfont.c"
```
Comment (drashna) [inline keyboards/crkbd/keymaps/default/rules.mk]: This doesn't do anything.  

It's not being processed, basically. 

You'd need something like this in the rules.mk here: 

```make
ifeq ($(strip $(LOCAL_GLCDFONT)), yes)
    OPT_DEFS += -DLOCAL_GLCDFONT
endif
```

**Comment by drashna** [comment]
Actually, wouldn't it be better to move the glcdfont.c to the "lib" folder, and just add it to the "SRC" value of the rules.mk in the keymap folders?

**Comment by foostan** [comment]
It's sounds good.
But 

- move the glcdfont.c to the `lib`
- add it to the `SRC` values of the rules.mk in the keymap folders

Compile error

```
Compiling: keyboards/crkbd/ssd1306.c                                                                [OK]
Compiling: keyboards/crkbd/lib/glcdfont.c                                                          keyboards/crkbd/lib/glcdfont.c:18:28: error: 'font' defined but not used [-Werror=unused-const-variable=]
 static const unsigned char font[] PROGMEM = {
                            ^~~~
```

Sorry I'm unfamiliar with C lang. Can you give me advice?

**Comment by drashna** [comment]
Oh, that's ... odd.  

It's that it's being defined "but not used".  Which is odd, as I compiled it on my system, and it works just fine.

But worst case, wrap it in `#ifdef SSD1306OLED` and `#endif` maybe.

File: keyboards/crkbd/lib/glcdfont.c [hunk 2]
```diff
@@ -14,8 +14,7 @@
 #endif
 
 // Standard ASCII 5x7 font
-
-static const unsigned char font[] PROGMEM = {
+const unsigned char font[] PROGMEM = {
```
Comment (foostan) [inline keyboards/crkbd/lib/glcdfont.c]: Compile success by remove `static`.

**Comment by drashna** [comment]
Just to make sure, this is good to go now?

**Comment by foostan** [comment]
yes please 😄

**Comment by drashna** [comment]
Done!  
Thanks!

**Pull request merged**